docs/fundamentals/runtime-libraries/snippets/System/Span/Overview/csharp/program.cs

```diff
@@ -20,18 +20,19 @@ private static void CreateSpanFromArray()
     {
         // <Snippet1>
         // Create a span over an array.
-        var array = new byte[100];
-        var arraySpan = new Span<byte>(array);
+        byte[] array = new byte[100];
+        Span<byte> arraySpan = new(array);
```

Comment:
You could replace the existing code with this to save an allocation. (As long as the Span doesn't get returned from the method)

```suggestion
        Span<byte> arraySpan = stackalloc byte[100]; 
```